examples/GroceryExample/readme.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+The goal of this Grocery example is to try and instrument the code. From this
+excersize we hope to discover and learn additional topics for discussions.
+
+We are focus only on the API side at the moment.  It is known that SDK
+implementation will likely affect how the API is designed, but we will make our
+best judgement to tolerate the situation at this time.
+
+## Topics for discussions
+
+- Need some kind of concrete LabelSet() in API.  Access to LabelSetSdk() is
+unavailable from API side.
+
+- It is inconvenient to have to pass a default(SpanContext) when we don't care
+about spans. Need additional prototypes to make SpanContext optional.
+
+- We create instrument with CreateInt64Counter() but it returns a generic
+Counter&lt;long&gt;. Seems like it should return a Int64Counter instead.
+
+- It's not allowed to new MeterProvider().  Thus, the only way to access is via
+the Default property. We can probably simplify MeterProvider.Default.GetMeter()
+to simply MeterProvider.GetMeter().
+
+- Bound counters does not allow adding more labels.  Ideally, we would bind
```

Comment:
yes this is something we need to solve. The bound counter in the original spec did not allow this. So bounds instruments were useable only if you know the key/value for *all* the labels, not just a part of them.

Reply:
We have new APIs soon.  Let's address it then.